project_one.html

```diff
@@ -201,6 +190,21 @@ <h2 class="about_h" >
           <hr />
         </div>
       </section>
+      <section class="contact">
+        <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I'd love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/" method="post">
+        <input type="text" name="user_name" id="name" placeholder="Enter your name" minlength="30" required />
+        <input type="email" name="user_email" id="email" placeholder="Enter your email" required />
+        <textarea name="message" id="textarea" minlength="500" cols="30" rows="5" required
+        placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Great job so far :+1:  However According to the project requirement: 

- The name field must accept up to 30 chars, which means, the Name input should have the `maxlength="30"` attribute instead of  `minlength`.
- The message field accepts up to 500 chars which means, the Message input should have the `maxlength="500"` attribute instead of  `minlength` attribute.

Reply:
Thank you sir for highlighting my mistakes. It is helpful for me